### DISCUSSION TITLE
Does it work on android?

### DISCUSSION THREAD

**Discussion by hazem22123** [GitSquared/edex-ui#1058] [Questions] [answered]
I tried to run it in termux but there is an error

bash: ./eDEX-UI-Linux-x86_64.AppleImage: cannot execute binary file: Exec format error

**Comment by Itai-Nelken** [comment]
> I tried to run it in termux but there is an error
> 
> bash: ./eDEX-UI-Linux-x86_64.AppleImage: cannot execute binary file: Exec format error

you are trying to run the x86_64 appimage on arm64, download the arm64 appimage.
also I'm pretty sure it won't work on termux, but I'm pretty sure it will work if you install a desktop using a app like 'andronix'.

**Comment by GitSquared** [comment, marked as answer]
eDEX won't work on Android, nor would it be usable if it did. Everything from the inner workings to the interface is made for desktop operating systems.

**Comment by 0x4248** [comment]
edex-ui uses electron and electron is really for desktop's

**Comment by ghost** [comment]
You might be able to get it to work on android by installing [UserLAnd](https://play.google.com/store/apps/details?id=tech.ula) and then building in that environment.
